osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/AbstractSchedulingConfig.java

```diff
@@ -58,6 +62,12 @@
     @Autowired
     private ApplicationContext applicationContext;
 
+    private int startAttemptCount = 0;
+
+    private final int maxStartAttemptCount = 10;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1170)